app.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask, render_template, request, redirect, url_for, flash, session
+from flask_sqlalchemy import SQLAlchemy
+from flask_bcrypt import Bcrypt
+from config import Config
+from models import db, User
+
+app = Flask(__name__)
+app.config.from_object(Config)
+
+# Initialize extensions
+db.init_app(app)
+bcrypt = Bcrypt(app)
+
+# Create tables explicitly during app initialization
+with app.app_context():
+    db.create_all()
+
+# Routes
+@app.route('/')
+def home():
+    return render_template('index.html')
+
+@app.route('/register', methods=['GET', 'POST'])
```

Comment:
Why are there two types of `method verb` ? I mean why its `GET` and `POST` both?